protobuf-c/plan.sh

```diff
@@ -0,0 +1,28 @@
+pkg_origin=core
+pkg_name=protobuf-c
+pkg_version=0.15
+pkg_dirname=protobuf-c-ddabb041b38ce0ab1fa8e91734e79b206556a661
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('Apache-2.0')
```

Comment:
As per https://github.com/protobuf-c/protobuf-c/blob/ddabb041b38ce0ab1fa8e91734e79b206556a661/LICENSE , no this is Apache 2.0 at this release.